views/default/event_calendar/personal_toggle_js.php

```diff
@@ -2,17 +2,13 @@
 
 // TODO: put the rest into a JS function
 
-$elgg_ts = time();
-$elgg_token = generate_action_token($elgg_ts);
-$tokens = "&__elgg_ts=$elgg_ts&__elgg_token=$elgg_token";
 ?>
 
 <script type="text/javascript">
 function event_calendar_personal_toggle(event_id, user_id) {
 
-	var link = "<?php echo elgg_get_site_url(); ?>action/event_calendar/toggle_personal_calendar?";
-	link += "user_id="+user_id+"&event_id="+event_id+"&other=true";
-	link += "<?php echo $tokens; ?>";
+	var link = "<?php echo elgg_get_site_url(); ?>action/event_calendar/toggle_personal_calendar";
+	link += elgg.security.addToken("?user_id="+user_id+"&event_id="+event_id+"&other=true");
```

Comment:
Even better would be to replace ```$.get()``` with ```elgg.action()``` which both:
 - Formats the URL (you won't have to add wwwroot manually)
 - Adds the tokens automatically

See http://learn.elgg.org/en/1.10/guides/javascript.html#ajax-helper-functions